ui/hooks/src/utils/media-utils.ts

```diff
@@ -172,19 +124,19 @@ export const saveMediaFile = async ({ name, content, isUrl, email }: ISaveMediaF
 };
 
 /**
- * Utility to get profile image versions with media url
+ * Utility to get profile image url
  */
-export const getProfileImageVersionsWithMediaUrl = (
+export const getProfileImageUrl = (
```

Comment:
can't this function be applied to all images? is profile image different?

Reply:
The parameter it takes and the one it returns are specifically related to the profile image. We are working with this type `AkashaProfileImageVersions`.

Reply:
The question is do we have to create a function for every typing? Can we use function overloading? is the return type different for other images?

Reply:
> is the return type different for other images?

Yes